assets/scripts/build_gallery.py

```diff
@@ -618,11 +773,13 @@ def main(**kwargs):
     # context_dir, context = _disable_progress_bars()
     context = None
 
-    gallery_info = build_gallery(
+    build_gallery(
+        only_combine=kwargs["only_combine"],
         include_core=not kwargs["no_core"],
         include_contrib=not kwargs["no_contrib"],
         ignore_suppress=kwargs["ignore_suppress"],
         ignore_only_for=kwargs["ignore_only_for"],
+        outfile_name=kwargs["outfile_name"],
         only_these_expectations=kwargs["args"],
         only_consider_these_backends=backends,
```

Comment:
Is `backends` what we will use in the pipeline to generate the gallery data file for each individual backend?